app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -176,6 +178,7 @@ class UrlInputFragment :
             clearView.visibility = View.VISIBLE
             searchViewContainer.visibility = View.GONE
         }
+        animateWordMark()
```

Comment:
This is a great idea. One drawback here is that the animation will also play when the app launches the first time. The bug description is not really clear about that - sorry! What UX was asking for is to only do this alpha animation if the user comes back to this screen after erasing the browsing session. In other situations (like the app start) there should no alpha animation be added.

Reply:
Thank you for this informations. I will do this changes.